css/style.css

```diff
@@ -105,6 +101,44 @@ header {
     left: 50%;
 }
 
+.contact-wrapper {
+    width: 100%;
+    background-color: #353535;
+}
+
+.section-header {
+    padding-top: 10px;
+    height: 250px;
+}
+
+.front-text {
+    color: #ffffff;
+    font-family: 'Muli', sans-serif;
+    font-weight: 700;
+    font-size: 4rem;
+    z-index: 2;
+    position: relative;
+    top: 55px;
+    text-transform: uppercase;
+}
+
+.shadow-text {
+    font-family: 'Covered By Your Grace', cursive;
+    color: #A9BCD0;
+    opacity: 0.3;
+    font-size: 7rem;
+    position: relative;
+    top: -173px;
+    z-index: 1;
```

Comment:
Not needed

Reply:
I have included a z-index here as Mike said it was good practice.